src/kemal/ext/response.cr

```diff
@@ -1,7 +1,7 @@
 class HTTP::Server::Response
   class Output
     def close
-      unless response.wrote_headers? && !response.headers.has_key?("Content-Range")
+      if !response.wrote_headers? && response.headers.has_key?("Content-Range")
```

Comment:
Is this a linting thing?

Reply:
Probably more so readability.  @Sija can confirm.

Reply:
@wontruefree Yes, it's related to double negation - `unless` with `!`.